src/apps/advanced-search/AdvancedSearchResults.tsx

```diff
@@ -88,7 +88,7 @@ const AdvancedSearchResult: React.FC<AdvancedSearchResultProps> = ({
   }, [data, page]);
 
   const shouldShowSearchResults = isLoading || (!isLoading && hitcount > 0);
-  const shouldShowResultHeadline = hitcount && !isLoading;
+  const shouldShowResultHeadline = !!(hitcount && !isLoading);
```

Comment:
I prefer casting to Boolean. I think it is easier to read. But that is a matter of taste.
```typescript
  const shouldShowResultHeadline = Boolean(hitcount && !isLoading);